scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -62,6 +88,32 @@ trait AndroidAppModule extends JavaModule {
    */
   def androidReleaseKeyPath: T[PathRef] = Task.Source(millSourcePath / androidReleaseKeyName())
 
+  /**
+   * Specifies the file format of lint report. Available file formats are "html", "xml", "txt" and "sarif".
+   */
+  def androidLintReportFmt: T[AndroidLintReportFormat.Value] = Task { AndroidLintReportFormat.Html }
+
+  /**
+   * Specifies the lint configuration XML file path. This allows setting custom lint rules or modifying existing ones.
+   */
+  def androidLintConfigPath: T[Option[PathRef]] = Task { None }
+
+  /**
+   * Specifies the lint baseline XML file path. This allows using a baseline to suppress known lint warnings.
+   */
+  def androidLintBaselinePath: T[Option[PathRef]] = Task { None }
+
+  /**
+   * Determines whether the build should fail if Android Lint detects any issues.
+   */
+  def abortOnError: Boolean = true
```

Comment:
should it be `androidLintAbortOnError` for the consistency with other API methods?